cmake/modules/FindLibsrtp.cmake

```diff
@@ -0,0 +1,79 @@
+#.rst:
+# FindLibsrtp
+# -------
+# Finds the libsrtp library
+#
+# This will define the following variables::
+#
+#   LIBSRTP_FOUND - system has libsrtp
+#   LIBSRTP_INCLUDE_DIRS - the libsrtp include directory
+#   LIBSRTP_LIBRARIES - the libsrtp libraries
+#
+# and the following imported targets::
+#
+#   libsrtp::libsrtp - The libsrtp library
+#
+
+# If target exists, no need to rerun find. Allows a module that may be a
+# dependency for multiple libraries to just be executed once to populate all
+# required variables/targets.
+if(NOT TARGET libsrtp::libsrtp)
+  if(ENABLE_INTERNAL_LIBSRTP)
+    include(cmake/scripts/common/ModuleHelpers.cmake)
+
+    set(MODULE_LC libsrtp)
+
+    SETUP_BUILD_VARS()
+
+    set(LIBSRTP_VERSION ${${MODULE}_VER})
+
+    set(CMAKE_ARGS -DBUILD_WITH_WARNINGS=OFF
+                   -DENABLE_OPENSSL=ON
+                   -DLIBSRTP_TEST_APPS=OFF)
+
+    set(patches "${CORE_SOURCE_DIR}/tools/depends/target/${MODULE_LC}/0001-Disable-build-with-warnings.patch"
+                "${CORE_SOURCE_DIR}/tools/depends/target/${MODULE_LC}/0002-Enable-OpenSSL.patch")
+
+    generate_patchcommand("${patches}")
+
+    BUILD_DEP_TARGET()
+  else()
+    # Populate paths for find_package_handle_standard_args
+    find_path(LIBSRTP_INCLUDE_DIR NAMES srtp2/srtp.h)
+    find_path(LIBSRTP_LIBRARY NAMES srtp2)
```

Comment:
This should be find_library presumably.

Reply:
Non-internal builds aren't tested, though will need to be if we want libtorrent as a dependency